pkg/cmd/gloo_client/root.go

```diff
@@ -1,20 +1,14 @@
-package cmd
+package gloo_client
```

Comment:
nit: don't use underscores in go package names (https://blog.golang.org/package-names describes package naming conventions)

i think `client` would be fine here (or `glooclient`)

Reply:
good catch, updated